Hazelcast.Net/Hazelcast.IO.Serialization/SerializationService.cs

```diff
@@ -349,13 +348,15 @@ protected internal ISerializerAdapter SerializerFor(int typeId)
             if (typeId <= 0)
             {
                 var index = IndexForDefaultType(typeId);
-                if (index < ConstantSerializersSize)
+                if (index < ConstantSerializersSize && index != -SerializationConstants.DefaultTypeSerializable)
                 {
                     return _constantTypeIds[index];
                 }
             }
+
```

Comment:
We don't use empty lines like this. I can see a couple of them below too. 
Please do not reformat the code unless we change the formatting rules.

Reply:
Removed

Reply:
Can you remove other spaces you added?

Reply:
Restored to original state